guides/user/kubernetes-v2/patch-manifest/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: single
+title:  "Patch Kubernetes Manifests"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.8" %}
+
+{% include toc %}
+
+This guide shows the basics of how to update a Kubernetes resource in place using the patch manifest stage for the [Kubernetes Provider V2](/setup/install/providers/kubernetes-v2) provider.
+
+There are a few steps:
+
+* [Specify which manifest to patch](#specify-the-resource-to-patch)
+
+* [Specify the patch content](#specify-your-patch-content)
+
+* [(Optional) Override artifacts in the patch content](#override-artifacts)
+
+  Optionally, you can override some artifacts (as fields) at run time (for
+    example, which Docker image to use.)
+
+* [(Optional) Override patch specific options](#specify-patch-options)
+
+## Specify the resource to patch
+
+Start by selecting the __Patch (Manifest)__ stage from the stage selector:
+
+{%
+  include
+  figure
+  image_path="./patch-manifest.png"
+%}
+
+To identify the Kubernetes resource to patch, specify the following required fields:
+
+* __Account__
+
+  The Spinnaker account that manages the Kubernetes cluster
+
+* __Namespace__
+
+  The Kubernetes namespace that your resource is located in.
+
+* __Kind__
+
+  The Kubernetes Kind of your resource e.g. deployment, service etc.
+
+* __Name__
+
+  The name of your Kubernetes resource.
+
+
+## Specify your patch content
+
+The patch content is similar to a manifest in the Deploy (Manifest) stage. However, unlike the deploy manifest, this does not have to be the full resource manifest but only the portion you want to patch.
+
+Depending on your needs, there is more than one way to specify the patch content:
+
+* [Statically: directly in the pipeline](#specify-patch-content-statically)
+* [Dynamically: bound at runtime using an artifact](#specify-patch-content-dynamically)
+
+
+### Specify patch content statically
+
+You can enter the patch content YAML by hand. For instance, if you want to patch your manifest to add a new label, you will specify the following:
+
+```yaml
+metadata:
+  labels:
+    foo: bar
+```
+
+{%
+  include
+  figure
+  image_path="./in-pipeline.png"
+  caption="Notice that by selecting __Text__ as the __Manifest Source__, we get
+  to enter the manifest YAML by hand."
+%}
+
+### Specify patch content dynamically
+
+Like the [Deploy (Manifest) stage](/guides/user/kubernetes-v2/deploy-manifest/index.md#specify-manifests-dynamically), you can also reference an artifact as the source if you are storing your patch content externally. The artifact must be a text file containing the patch content.
```

Comment:
The content of the text file is the same yml format as the statically specified patch above?

Reply:
Yes